snmp/datadog_checks/snmp/data/default_profiles/peplink.yaml

```diff
@@ -0,0 +1,164 @@
+extends:
+  - _base.yaml
+  - _generic-host-resources-base.yaml
+  - _generic-if.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.23695
+  - 1.3.6.1.4.1.23695.*
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "peplink"
+
+metrics:
+  - MIB: PEPLINK-DEVICE
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.3.1.0
+  - MIB: PEPLINK-DEVICE
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.3.2.0
+  - MIB: PEPLINK-DEVICE
+    symbol:
+      name: memory.usage
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.3.3.0
+  - MIB: PEPLINK-DEVICE
+    symbol:
+      name: peplink.deviceTemperatureCelsius
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.4.4.1.0
+  - MIB: PEPLINK-DEVICE
+    symbol:
+      name: peplink.deviceTemperatureFahrenheit
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.4.4.2.0
+  - MIB: PEPLINK-DEVICE
+    table:
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.4.1
+      name: devicePSUTable
+    symbols:
+      - name: peplink.devicePSUPercentage
+        OID: 1.3.6.1.4.1.23695.200.1.1.1.4.1.1.5
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.1.1.1
+          name: devicePSUId
+        tag: device_psu_id
+      - column:
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.1.1.2
+          name: devicePSUStatus
+        tag: device_psu_status
+        mapping:
+          0: error
+          1: 'on'
+  - MIB: PEPLINK-DEVICE
+    table:
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.4.2
+      name: deviceFanTable
+    symbols:
+      - name: peplink.deviceFanSpeed
+        OID: 1.3.6.1.4.1.23695.200.1.1.1.4.2.1.3
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.2.1.1
+          name: deviceFanId
+        tag: device_fan_id
+      - column:
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.2.1.2
+          name: deviceFanStatus
+        tag: device_fan_status
+        mapping:
+          0: error
+          1: 'on'
+  - MIB: PEPLINK-DEVICE
+    table:
+      OID: 1.3.6.1.4.1.23695.200.1.1.1.4.3
+      name: devicePowerSourceTable
+    symbols:
+      - name: peplink.devicePowerSource
+        constant_value_one: true
+    metric_tags:
+      - column:
+          name: devicePowerSourceId
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.3.1.1
+        tag: device_power_source_id
+      - column:
+          name: devicePowerSourceName
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.3.1.2
+        tag: device_power_source_name
+      - column:
+          OID: 1.3.6.1.4.1.23695.200.1.1.1.4.3.1.3
+          name: devicePowerSourceStatus
+        tag: device_power_source_status
+        mapping:
+          0: no_cable_detected
+          1: connected
+  # - MIB: PEPLINK-BALANCE-MIB # TODO: Handle string for "up"/"down"
+  #   symbol:
+  #     name: peplink.balLanStatus
+  #     OID: 1.3.6.1.4.1.23695.1.1.1.6.1
+  #     format: TODO_handle_conversion_to_one
```

Comment:
this should be the equivalent of `constant_value_one: true`

Reply:
What do you mean?

Reply:
the `conversion: to_one` on kentik's side is the equivalent of `constant_value_one: true` on our side
[example with this metric](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/brother-net-printer.yaml#L30-L37)

Reply:
But isn't that only for table metrics at the moment? I don't think we have a way to support this with scalar metrics

Reply:
Right now, it's not possible to use `conversion_to_one` for a scalar metrics.